python/ray/tune/tests/test_trial_relative_logdir.py

```diff
@@ -1,322 +1,44 @@
 import os
-import shutil
 import sys
-import tempfile
-import unittest
-from pathlib import Path
-
-import pandas as pd
-import pandas.testing as pd_testing
 import pytest
 
-import ray
-from ray import tune
-from ray.air._internal.checkpoint_manager import CheckpointStorage, _TrackedCheckpoint
 from ray.tune.experiment import Trial
 
+from ray.train.tests.util import mock_storage_context
 
-def train(config):
-    tune.report(metric1=2, metric2=3)
-
-
-class TrialRelativeLogdirTest(unittest.TestCase):
-    def setUp(self):
-        # Make the data frame equality assertion available
-        # via `assertEqual`.
-        self.addTypeEqualityFunc(pd.DataFrame, self.assertDataFrameEqual)
-        if ray.is_initialized:
-            ray.shutdown()
-        ray.init(num_cpus=1, num_gpus=0)
-
-        tune.register_trainable("rel_logdir", train)
-
-    def tearDown(self):
-        ray.shutdown()
-
-    # Compare pandas data frames inside of the test suite.
-    def assertDataFrameEqual(self, df1, df2, msg):
-        try:
-            pd_testing.assert_frame_equal(df1, df2)
-        except AssertionError as e:
-            raise self.failureException(msg) from e
-
-    def testDotsInLogdir(self):
```

Comment:
These will be tested by new unit tests: https://github.com/ray-project/ray/pull/39023